dags/mag_and_friends.py

```diff
@@ -385,25 +395,24 @@ def create_dag():
                                 task_group=current_tasks
                             )                        
 
-                            copy_results_from_cluster = cluster_manager.copyFromCluster(
-                                tailStudyPath, 
-                                "out", 
-                                studyPath,
+                            copy_metatdenovo_results_from_cluster = cluster_manager.copyFromCluster(
+                                f"{tailStudyPath}/out/metatdenovo_out", 
+                                "summary_tables", 
+                                f"{studyPath}/out/metatdenovo_out",
                                 gzip=False,
-                                task_id = "copy_results_from_cluster",
+                                task_id = "copy_metatdenovo_results_from_cluster",
                                 task_group=current_tasks
                             )
 
                             # this should make rda files for the r package via a dedicated Rscript
-                            @task(task_group=current_tasks)
-                            def post_process_results():
-                                # TODO make sure the out path is correct
-                                cmd = f"Rscript /data/MicrobiomeDB/mbio_airflow_dags/bin/mag_postProcessing.R {studyName} {studyPath}/out"
-                                return(cmd)
+                            post_process_results = BashOperator(
+                                task_id="post_process_results",
+                                bash_command=f"Rscript /data/MicrobiomeDB/mbio_airflow_dags/bin/mag_postProcessing.R {studyName} {studyPath}/out",
```

Comment:
What would be good times to use hard coded file path? Just generally curious about this. If this is DAG is to be deployed else where?

Reply:
as much as possible should be avoided, but i didnt immediately think of a good way to avoid it here and was considering it a secondary priority for now